src/App/Patterns/content/Forms/index.js

```diff
@@ -9,32 +9,45 @@ import RadioButton from "@components/FormComponents/Radio";
 
 import LoginForm from "../../components/LoginForm/index";
 
-import { ComponentPreview } from "~/src/App/docutils";
+import { EditableComponentPreview } from "~/src/App/docutils";
 import { guidelinesElementsList, formsShowCase } from "./constants";
 import { hintTextExpander } from "@src/scripts/main";
 
 const radioOptions = [
     {
         label: "Option 1",
         id: "guidelines-options-1",
-        checked: true
+        checked: true,
     },
     {
         label: "Option 2",
-        id: "guidelines-options-2"
+        id: "guidelines-options-2",
     },
     {
         label: "Option 3",
-        id: "guidelines-options-3"
-    }
+        id: "guidelines-options-3",
+    },
 ];
 
 const Overview = () => (
     <section>
         <h2 id="overview">Overview</h2>
 
-        <p>A form can include different type of user input components that can be arranged in different ways depending on your specific context and needs. You might for example design a form for a user to sign up for or log into an account, register for a service, reconfigure settings, take a survey, purchase a product or provide feedback.</p>
-        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={formsShowCase} noOptions removeOuterTag/>
+        <p>
+      A form can include different type of user input components that can be
+      arranged in different ways depending on your specific context and needs.
+      You might for example design a form for a user to sign up for or log into
+      an account, register for a service, reconfigure settings, take a survey,
+      purchase a product or provide feedback.
```

Comment:
Why are these several lines? Looks a bit weird? Or is this also something you really are into, like dangling commas and bread with butter and honey?

Reply:
I agree with you,
I think my ESlint got a mental breakdown and auto-"_fixed_" things I never asked for,
I'll try to revert this